src/main/java/frc/robot/commands/DefaultIntakeCommand.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.IntakeConstants;
+import frc.robot.subsystems.Input;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class DefaultIntakeCommand extends Command {
+
+  private boolean fullPower = false;
+  private IntakeSubsystem mIntake;
+  /** Creates a new DefaultIntakeCommand. */
+  public DefaultIntakeCommand(IntakeSubsystem intake) {
+    mIntake = intake;
+    addRequirements(intake);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    mIntake.stopMotor();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (Input.getFullPower()) {
+      fullPower = !fullPower;
+    }
+    if (mIntake.pieceInIntake()) {
+      mIntake.stopMotor();
+    }
+    if (fullPower || (Input.getIntake() && !mIntake.pieceInIntake())) {
```

Comment:
Ooh this wasn't exactly what I meant lol. I think the entire execute function should look a little like this:

```
if (Input.getIntake && (!mIntake.pieceInIntake() || fullPower)) {
    mIntake.intakeAtSpeed(IntakeConstants.INTAKE_SPEED);
}
else {
    mIntake.stopMotor()
}

if (Input.getFullPower()) {
    fullPower = !fullPower
}
```